wgsl/index.bs

```diff
@@ -11111,7 +11211,7 @@ textureGather(t: texture_depth_cube_array, s: sampler, coords: vec3<f32>, array_
   Only applies to non-depth textures.
   <br>The index of the channel to read from the selected texels.
   <br>When provided, the `component` expression must be either:
-    * a `const_expression` expression (e.g. `1`).<br>
+    * a [=creation-time expression=] (e.g. `1`).<br>
     * a name of a [=module scope|module-scope=] [=let declaration=]
```

Comment:
I think this list is now unnecessary as we should just require a "creation-time expression" (module scope let is not a thing any more). Same for the `offset` parameter in the other texture builtins.

Reply:
Good call, module scope let's aren't valid anymore either!